tests/wallet/test_singleton_lifecycle_fast.py

```diff
@@ -31,25 +31,25 @@
 P2_SINGLETON_MOD_HASH = P2_SINGLETON_MOD.get_tree_hash()
 
 ANYONE_CAN_SPEND_PUZZLE = Program.to(1)
-ANYONE_CAN_SPEND_WITH_PADDING_PUZZLE_HASH = Program.to(binutils.assemble("(a (q . 1) 3)")).get_tree_hash()
+ANYONE_CAN_SPEND_WITH_PADDING_PUZZLE_HASH = Program.to(binutils.assemble("(a (q . 1) 3)")).get_tree_hash()  # type: ignore[no-untyped-call] # noqa: E501
 
 POOL_REWARD_PREFIX_MAINNET = bytes32.fromhex("ccd5bb71183532bff220ba46c268991a00000000000000000000000000000000")
 
 MAX_BLOCK_COST_CLVM = int(1e18)
 
 
 class PuzzleDB:
-    def __init__(self):
-        self._db = {}
+    def __init__(self) -> None:
+        self._db: Dict[bytes32, Program] = {}
 
-    def add_puzzle(self, puzzle: Program):
+    def add_puzzle(self, puzzle: Program) -> None:
         self._db[puzzle.get_tree_hash()] = Program.from_bytes(bytes(puzzle))
 
     def puzzle_for_hash(self, puzzle_hash: bytes32) -> Optional[Program]:
         return self._db.get(puzzle_hash)
 
 
-def from_kwargs(kwargs, key, type_info=Any):
+def from_kwargs(kwargs: Dict[str, Any], key: str, type_info: Any = Any) -> Any:
```

Comment:
this looks fishy:
```
type_info: Any = Any
```

Should it not be `Type[Any]`?

Reply:
I tried that originally but I got `error: Incompatible default for argument "type_info" (default has type "object", argument has type "type[Any]")  [assignment]`. Please note this is currently not even used (as noted in the function's docstring) so I deemed `Any` to be adequate at this point.

Reply:
could we just delete the parameter instead?

Reply:
Sure. PR #17375.